Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml

```diff
@@ -59,13 +59,12 @@ ParserQuery: |
         [
             "0x1", "RegistryValueRead"
             , "0x10", "MetadataModified"
```

Comment:
Notice that per your table, 0x10 is RegistryKeyNotify

Reply:
Updated the parser